src/Compiler/Service/FSharpCheckerResults.fs

```diff
@@ -2089,6 +2089,36 @@ type internal TypeCheckInfo
                 Trace.TraceInformation(sprintf "FCS: recovering from error in GetSymbolUseAtLocation: '%s'" msg)
                 None)
 
+    member _.GetSymbolUsesAtLocation(line, lineStr, colAtEndOfNames, names) =
+        DiagnosticsScope.Protect
+            range0
+            (fun () ->
+                let declItemsOpt =
+                    GetDeclItemsForNamesAtPosition(
+                        None,
+                        Some names,
+                        None,
+                        None,
+                        line,
+                        lineStr,
+                        colAtEndOfNames,
+                        ResolveTypeNamesToCtors,
+                        ResolveOverloads.Yes,
+                        None,
+                        (fun () -> [])
+                    )
+
+                match declItemsOpt with
+                | None -> List.empty
+                | Some (items, denv, _, m) ->
+                    items
+                    |> List.map (fun item ->
+                        let symbol = FSharpSymbol.Create(cenv, item.Item)
+                        symbol, item.ItemWithInst, denv, m))
+            (fun msg ->
+                Trace.TraceInformation(sprintf "FCS: recovering from error in GetSymbolUsesAtLocation: '%s'" msg)
```

Comment:
Any idea when this can actually happen? Just curious.

Reply:
Not really, the old code had it. I guess it is a guard against something unexpected like a null pointer or something.